examples/softirq-latency-net-rx.bpf.c

```diff
@@ -4,14 +4,21 @@
 
 // Loosely based on https://github.com/xdp-project/xdp-project/blob/master/areas/latency/softirq_net_latency.bt
 
-// 20 buckets for latency, max range is 0.5s .. 1.0s
-#define MAX_LATENCY_SLOT 21
+// 30 buckets for latency, max range is 0.5s .. 1.0s
+#define MAX_LATENCY_SLOT 31
```

Comment:
Is nanosecond precision worth 10 extra buckets? Looking at the output here, perhaps the answer is yes:

* https://bugzilla.redhat.com/show_bug.cgi?id=1795049

Reply:
You have this info in the commit already.